bower.json

```diff
@@ -8,6 +8,6 @@
   "devDependencies": {
     "jquery": "~1.7",
     "jasmine-ajax": "~1.3.1",
-    "jasmine-jquery": "~1.5.2"
+    "jasmine-jquery": "~2.1.1"
```

Comment:
I am uncertain if this has to be rolled back to version 1.7.0 or perhaps a version 2 before they added the dependency to jquery 2 in jasmine-jquery library.

See discussion https://github.com/velesin/jasmine-jquery/issues/267